src/test/javascript/spec/component/shared/metis/post/post.component.spec.ts

```diff
@@ -27,7 +29,7 @@ describe('PostComponent', () => {
     beforeEach(() => {
         return TestBed.configureTestingModule({
             imports: [],
-            declarations: [PostComponent, MockPipe(HtmlForMarkdownPipe)],
+            declarations: [PostComponent, MockPipe(HtmlForMarkdownPipe), MockComponent(PostHeaderComponent), MockComponent(PostFooterComponent)],
             schemas: [NO_ERRORS_SCHEMA],
```

Comment:
It looks like you might have forgotten to remove the NO_ERRORS_SCHEMA from this file, or is there another reason for not doing it?

Reply:
totally true, I forgot it. I removed it, tests still pass. Thanks for the hint!